docs/FAQ.md

```diff
@@ -46,11 +188,27 @@ engine components you desire, provide any necessary configurations, and build
 an instance of the loader. After that, you configure your system boot to launch
 the loader.
 
+## 7. Where do I start if I want to help with contributions?
 
-## 4. Where do I start if I want to help with contributions?
-
-The [TrenchBoot Blueprints](
-    https://github.com/TrenchBoot/documentation/tree/master/blueprints)
+The [TrenchBoot Blueprints](./blueprints/index.md)
```

Comment:
Reword:
"[TrenchBoot Blueprints](./blueprints/index.md) collect product feature requests. Check to see if there is an existing blueprint that addresses your feature request. You may also submit a blueprint via pull request to suggest features for implementation."

Edited-by: Amanda Ferguson [af.jenks@gmail.com](mailto:af.jenks@gmail.com)